### PR TITLE
Show amount of new proposals since last timeline visit 

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

**Comment by samuveth** [comment]
@bonustrack I'm not to sure about the calculation of unseen proposals with indexOf.

**Comment by bonustrack** [comment]
@samuveth I think it was fine before commit ae3a7ea

**Comment by bonustrack** [comment]
We need show the counter if the user dont have last seen stored in localStorage too

**Comment by bonustrack** [comment]
On mobile the counter appear on another line, can you fix that? Screenshot_20210428-120226_Chrome

**Comment by samuveth** [comment]
@bonustrack looks a bit weird like this, maybe hide it on mobile?
image

**Comment by bonustrack** [comment]
@samuveth I think it's fine like this

File: src/views/Home.vue [hunk 1]
```diff
@@ -2,7 +2,16 @@
   <div>
     <div class="text-center mb-4 mx-auto">
       <Container class="d-flex flex-items-center">
-        <div class="flex-auto text-left">
+        <div class="flex-auto text-left d-flex">
+          <router-link :to="{ name: 'timeline' }">
+            <UiButton class="mr-2" style="width: 135px">
```
Comment (bonustrack) [inline src/views/Home.vue:7]: Can you try like this?
```suggestion
            <UiButton class="mr-2 no-wrap">
```

Comment (samuveth) [reply src/views/Home.vue]: Yeah that's better

**Review by bonustrack — APPROVED** [review]

**Pull request merged**